sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Generated/Models/UpdateModelSettingsRequest.Serialization.cs

```diff
@@ -8,9 +8,9 @@
 using System.Text.Json;
 using Azure.Core;
 
-namespace Azure.Iot.TimeSeriesInsights
+namespace Azure.Iot.TimeSeriesInsights.Models
```

Comment:
In ADT, we decided that even models should belong to the same namespace as the core implementation. So we'll probably need to follow the same pattern here.

Reply:
Let's chat about this offline.